docs/section11/Rule_11-7.md

```diff
@@ -0,0 +1,47 @@
+# Service_Water_Heating - Rule 11-7  
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-7  
+
+**Rule Description:** Except in buildings that will have no service water heating loads,  the service water heating systems type in the baseline building design shall be as specified in Table G3.1.1-2 for each building area type in the proposed design. 
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a & b
+
+**Evaluation Context:** B-RMD each SWH Equipment
+**Data Lookup:**   
+- **Table G3.1.1-2(shw_bat)**
+**Function Call:** 
+- **get_component_by_id**
+- **get_SHW_equipment_associated_with_each_SWH_bat**  
+- **get_SHW_types_and_SHW_use**
+- **get_SHW_equipment_type**  
+
+**Applicability Checks:**
+- Each SHW use type is applicable if there are SHW loads
+- use get_SHW_equipment_associated_with_each_SWH_bat to get the SHW use types and SHW use in the building: `shw_bats_and_equip_dict = get_SHW_equipment_associated_with_each_SWH_bat(B_RMD)`
+
+- call the function get_SHW_types_and_SHW_use for the proposed: `shw_bat_and_use_dict_p = get_SHW_types_and_SHW_use(P_RMD)`
+- look through each of the SHW bats in the building: `for shw_bat in shw_bat_and_use_dict_p:`
+  - check if there are SHW loads in this use type, continue to rule logic: `if len(shw_bat_and_use_dict_p[shw_bat]) > 0: CONTINUE TO RULE LOGIC`
+  - otherwise, rule is not applicable for this shw_bat: `else: NOT_APPLICABLE`
+
+  ## Rule Logic: 
+  - look at each SWH Equipment in the shw_bat - there should only be one, but this is checked in rule 11-8: `for swh_equip_id in shw_bats_and_equip_dict[shw_bat]["SHWHeatingEq"]:`
```

Comment:
to confirm, the plan is to evaluate each SWH Equipment and not raise an error here because an error will be raised in rule 11-8 and we don't need to duplicate? If so, this could be reworded for clarity so that the dev team knows we are not trying to force exactly 1 ServiceWaterHeatingEquipment here